db/migrate/20190806165231_add_braintree_device_data_to_payment.rb

```diff
@@ -0,0 +1,7 @@
+class AddBraintreeDeviceDataToPayment < ActiveRecord::Migration[5.1]
+  def change
+    change_table :spree_payments do |t|
+      t.text :braintree_device_data
+    end
```

Comment:
we should keep things consistent and use a one line migration here since we're just adding a column instead of altering the data types:
`add_column :spree_payments, :braintree_device_data, :text`

updating here will updated the migration being loaded into engine_storefront